jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py

```diff
@@ -29,8 +29,8 @@ def __init__(self):
         self.bridge = CvBridge()
         self.smach_state_list = {}  # for status list
         self.smach_state_subject = {}  # for status subject
-        self.sender_address = "tsukamoto@jsk.imi.i.u-tokyo.ac.jp"
-        self.receiver_address = "tsukamoto@jsk.imi.i.u-tokyo.ac.jp"
+        self.sender_address = rospy.get_param("~sender_address")
```

Comment:
@tkmtnt7000 thank you, please add error checking code, when no param is set, use default value or raise errors

Reply:
Sorry for late.
I add error checking code(https://github.com/k-okada/jsk_robot/pull/71/commits/354bd5a58d9acbadb820a140f35e4cecae8b5aca)